packages/app-bridge/src/react/usePrivacySettings.ts

```diff
@@ -7,7 +7,21 @@ import type { PrivacySettings } from '../types/PrivacySettings';
 import { AppBridgeBlock } from '../AppBridgeBlock';
 
 export const usePrivacySettings = (appBridge: AppBridgeBlock) => {
-    const [privacySettings, setPrivacySettings] = useState<PrivacySettings>(appBridge.getPrivacySettings());
+    const [privacySettings, setPrivacySettings] = useState<PrivacySettings>({
+        assetDownloadEnabled: false,
+        assetViewerEnabled: false,
+    });
+
+    useEffect(() => {
+        const fetchPrivacySettings = async () => {
+            const privacySettings = await appBridge.api({
+                name: 'getPrivacySettings',
+                payload: { portalId: appBridge.context('portalId').get() },
+            });
+            setPrivacySettings(privacySettings);
+        };
+        fetchPrivacySettings();
```

Comment:
Optional.
To me this is a bit more readable, as fetchPrivacySettings does not imply that it also sets the state. `fetchPrivacySettings` could also potentially be moved to a helper function.

```suggestion
        const fetchPrivacySettings = async () => {
            return await appBridge.api({
                name: 'getPrivacySettings',
                payload: { portalId: appBridge.context('portalId').get() }
            });
        };
        setPrivacySettings(await fetchPrivacySettings());
```

Reply:
True it would be better readable, but it isn't that easy as `Effect callbacks are synchronous to prevent race conditions. Put the async function inside`, so I will leave it as it is as its easier to read than changing this